s32/mcux/devices/S32Z270/S32Z270_features.h

```diff
@@ -90,5 +98,17 @@
 #define FSL_FEATURE_FLEXCAN_HAS_ENHANCED_RX_FIFO (0)
 /* @brief Does not support Supervisor Mode (bitfield MCR[SUPV]. */
 #define FSL_FEATURE_FLEXCAN_HAS_NO_SUPV_SUPPORT (1)
+/* @brief Does not support Memory Error Control (bitfield MECR[HANCEI]. */
+#define FSL_FEATURE_FLEXCAN_HAS_NO_HANCEI_SUPPORT (1)
+/* @brief Does not support Memory Error Control (bitfield MECR[FANCEI]. */
+#define FSL_FEATURE_FLEXCAN_HAS_NO_FANCEI_SUPPORT (1)
```

Comment:
ERRSR_FANCEI masks don't need to be deleted?

Reply:
yes, don't need to be deleted them.